pkg/iac-providers/kustomize/v3/load-file.go

```diff
@@ -0,0 +1,11 @@
+package kustomizev3
+
+import (
+	"github.com/accurics/terrascan/pkg/iac-providers/kustomize/commons"
+	"github.com/accurics/terrascan/pkg/iac-providers/output"
+)
+
+// LoadIacFile is not supported for kustomize. Only loading directories that have kustomization.y(a)ml file are supported
+func (k *KustomizeV3) LoadIacFile(absRootPath string, options map[string]interface{}) (allResourcesConfig output.AllResourceConfigs, err error) {
+	return commons.NewKustomizeDirectoryLoader(absRootPath, options, true, versionSuffix).LoadIacFile()
```

Comment:
since file loading is not supported, should we return `nil, commons.ErrLoadIacFileNotSupported` instead of creating a new KustomizeDirectoryLoader?